Source/Core/Matrix2.js

```diff
@@ -57,13 +59,8 @@ define([
      */
     Matrix2.pack = function(value, array, startingIndex) {
         //>>includeStart('debug', pragmas.debug);
-        if (!defined(value)) {
-            throw new DeveloperError('value is required');
-        }
-
-        if (!defined(array)) {
-            throw new DeveloperError('array is required');
-        }
+        Check.typeOf.object(value, 'value');
+        Check.defined(array, 'array');
```

Comment:
Just to be sure, should `Check` have a `Check.typeOf.array` that uses `isArray` or is it not there to avoid problems with Arrays vs. TypedArrays?